### PR TITLE
Fix mistakes

### PR DESCRIPTION
**Pull request by z1ne2wo**

### CODE REVIEW

File: source/api-email-validation.rst [hunk 1]
```diff
@@ -143,7 +143,7 @@ did_you_mean             string       (Optional) Null if nothing, however if a p
 is_disposable_address    boolean      If the domain is in a list of disposable email addresses, this will be appropriately categorized
 is_role_address          boolean      Checks the mailbox portion of the email if it matches a specific role type ('admin', 'sales', 'webmaster')
 reason                   array        List of potential reasons why a specific validation may be unsuccessful.
-result                   string       Either ``deliverable``, ``undeliverable``, ``catch_all`` or ``unknown``. Please see the Result Types section below for details on each result type.
+result                   string       Either ``deliverable``, ``undeliverable``, ``do_not_send``, ``catch_all`` or ``unknown``. Please see the Result Types section below for details on each result type.
 risk                     string       ``high``, ``medium``, ``low``, or ``unknown`` Depending on the evaluation of all aspects of the given email.
```
Comment (z1ne2wo) [inline source/api-email-validation.rst:147]: Maybe section _Email Validation V4_ in [user_manual.rst](https://github.com/mailgun/documentation/blob/master/source/user_manual.rst) needs to be updated too -- missed `catch_all` for _result_ and `unknown` for _risk_:
Screenshot 2021-12-27 at 03 31 04

**Comment by thrawn01** [comment]
Thanks for the PR! I've asked for someone from the validation API team will review this change and your comment.

**Review by jeffreyjurgajtis — APPROVED** [review]

File: source/api-email-validation.rst (hunk 1 above)
Comment (jeffreyjurgajtis) [reply source/api-email-validation.rst:147]: Good catch. Yes, it would be great to update the user manual with the `catch_all` result and `unknown`

**Review by mkbond — APPROVED** [review]
LGTM! Thanks for taking the time to do this 🙂

**Pull request merged**